projects/igniteui-angular/src/lib/tabs/tabs/tab-header.component.ts

```diff
@@ -115,9 +115,11 @@ export class IgxTabHeaderComponent extends IgxTabHeaderDirective implements Afte
 
     /** @hidden @internal */
     public ngOnDestroy(): void {
-        this.ngZone.runOutsideAngular(() => {
-            this._resizeObserver.disconnect();
-        });
+        if (this._resizeObserver) {

```

Comment:
Wouldn't it be simpler just to do a check before calling `disconnect()`?
```suggestion
    /** @hidden @internal */
    public ngOnDestroy(): void {
          this.ngZone.runOutsideAngular(() => {
              this._resizeObserver?.disconnect();
          });
    }

```

Reply:
One more thing - `master` currently tracks `12.1.x`. If you want the fix to appear in a patch in `12.0.x` then you need to branch from `12.0.x` cherry-pick your changes and create an additional PR against `12.0.x`.